### PR TITLE
Fix/ AgendaList dayPress

### PR DESCRIPTION
**Pull request by lidord-wix**
Update the agenda position after pressing on any date (and not only while pressing date with a content in the agenda)
WOAUILIB-2119

### CODE REVIEW

File: example/src/screens/expandableCalendar.tsx [hunk 1]
```diff
@@ -190,6 +190,7 @@ export default class ExpandableCalendarScreen extends Component<Props> {
         <AgendaList
           sections={ITEMS}
           renderItem={this.renderItem}
+          // scrollToFutureEvents
```
Comment (Inbal-Tish) [inline example/src/screens/expandableCalendar.tsx:193]: Naming is a bit confusing. How about something like 'scrollToNextMarkedDate' or 'scrollToNextEvent'?
Also, You need to adjust the example to show the feature. I don't see it working in the example...

File: src/expandableCalendar/agendaList.tsx [hunk 2]
```diff
@@ -37,6 +50,8 @@ interface Props extends SectionListProps<any, DefaultSectionT> {
   avoidDateUpdates?: boolean;
   /** offset scroll to section */
   viewOffset?: number;
+  /** enable scrolling the agenda list to the next date with events while pressing a date without events */
```
Comment (Inbal-Tish) [inline src/expandableCalendar/agendaList.tsx:53]: Will skip empty dates? Or scroll to them if they are the next date with content?

File: src/expandableCalendar/agendaList.tsx [hunk 3]
```diff
@@ -86,10 +101,11 @@ class AgendaList extends Component<Props> {
 
   componentDidMount() {
     const {date} = this.props.context;
+    const {scrollToFutureEvents} = this.props;
     if (date !== this._topSection) {
       setTimeout(() => {
-        const sectionIndex = this.getSectionIndex(date);
-        if (sectionIndex) {
+        const sectionIndex = scrollToFutureEvents ? this.getNextSectionIndex(date) : this.getSectionIndex(date);
```
Comment (Inbal-Tish) [inline src/expandableCalendar/agendaList.tsx:107]: We should move these few lines to a method as it repeats on didUpdate

File: example/src/screens/expandableCalendar.tsx (hunk 1 above)
Comment (lidord-wix) [reply example/src/screens/expandableCalendar.tsx]: I updated the prop name.
The example works for me, try to uncomment this line and press on the next date in the weekCalendar

File: src/expandableCalendar/agendaList.tsx (hunk 2 above)
Comment (lidord-wix) [reply src/expandableCalendar/agendaList.tsx]: will scroll to the next date with content.. I changed the description :)

File: src/expandableCalendar/agendaList.tsx (hunk 3 above)
Comment (lidord-wix) [reply src/expandableCalendar/agendaList.tsx]: I moved it into `scrollToSection`

File: example/src/screens/expandableCalendar.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply example/src/screens/expandableCalendar.tsx]: I did... But when I press the 3rd of the month it doesn't scroll to the 4th, where there is content...

Comment (Inbal-Tish) [reply example/src/screens/expandableCalendar.tsx]: Sorry. It works now. Probably cache...

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**